src/poliastro/neos.py

```diff
@@ -0,0 +1,141 @@
+'''Code related to NEOs
+
+All new functions coded as part of SOCIS 2017 proposal
+
+'''
+import re
+from bs4 import BeautifulSoup
+import requests
+
+import astropy.units as u
+
+from poliastro.twobody.orbit import Orbit
+from poliastro.bodies import Sun
+from poliastro.twobody.angles import M_to_nu
+
+def get_orbit_from_spk_id(spk_id, epoch=None):
+    """Return `~poliastro.twobody.orbit.Orbit` given a SPK-ID.
+
+    Retrieve info from NASA NeoWS API, and therefore
+    it only works with NEAs (Near Earth Asteroids)
+
+    Parameters
+    ----------
+    spk_id : str
+        SPK-ID number, which is given to each body by JPL.
+
+    Returns
+    -------
+    orbit : ~poliastro.twobody.orbit.Orbit
+        NEA orbit..
+
+    """
+    url = 'https://api.nasa.gov/neo/rest/v1/neo/'
+    payload = {'api_key' : 'DEMO_KEY'}
+    try:
+        response = requests.get(url + spk_id, params=payload)
+        if response.status_code == 200:
+            orbital_data = response.json()['orbital_data']
+
+            attractor = Sun
+            a = float(orbital_data['semi_major_axis']) * u.AU
+            ecc = float(orbital_data['eccentricity']) * u.one
+            inc = float(orbital_data['inclination']) * u.deg
+            anl = float(orbital_data['ascending_node_longitude']) * u.deg
+            parg = float(orbital_data['perihelion_argument']) * u.deg
+            M = float(orbital_data['mean_anomaly']) * u.deg
+            nu = M_to_nu(M.to(u.rad), ecc)
+
+            if epoch is None:
+                return Orbit.from_classical(attractor, a, ecc, inc,
+                                        anl, parg, nu)
+            else:
+                return Orbit.from_classical(attractor, a, ecc, inc,
+                                        anl, parg, nu, epoch)
+        elif response.status_code == 404:
+            print('\nError 404. Object ID does not exists, or it does not match a NEA.')
+        else:
+            print('\nForbidden or unauthorized request: ', response.status_code)
+        return None
+    except requests.exceptions.RequestException:
+        print('\nSome kind of connection error ocurred, please try later.')
+        return None
+    except KeyError:
+        print('Internal error, please report it to Poliastro team')
```

Comment:
If you capture the error like this, there is no traceback and nothing to report to the team :) Better don't catch it and people will come to open an issue.

It's important to keep in mind what the intended audience of poliastro is, at least now: if they could download and install poliastro, especially the development version, they will understand how to open an issue in the repository. I admit this does not work for everybody, but let's create a developer-friendly tool first and later we can add a user-friendly layer above.